src/backend/core/services/converter_services.py

```diff
@@ -34,7 +34,7 @@ def auth_header(self):
         # Note: Yprovider microservice accepts only raw token, which is not recommended
         return settings.Y_PROVIDER_API_KEY
 
-    def convert_markdown(self, text):
+    def convert(self, text):
```

Comment:
Maybe the docstring of this method should be updated accordingly ?

Reply:
fair enough